src/Controls/Maps/src/HandlerImpl/Circle.Impl.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Maps;
+
+namespace Microsoft.Maui.Controls.Maps
+{
+	public partial class Circle : ICircleMapElement
```

Comment:
I'd say we don't follow the same organizational structure here that we currently are on core. That was very much a placeholder organization until we found our footing. Have we found our footing? :-p

Most of this seems simple enough that I think we can just condense it all together into the same files

If we need to add any platform specific partials for these then I think at that point we can just create a folder for the type. 

If you want to maintain the split here with Core.Impl vs BO then I'd say just create a folder for each type and put the files close to each other that way